packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/breadcrumbs.twig

```diff
@@ -1,14 +1,23 @@
-
-<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
-  <nav class="ma__breadcrumbs__container">
-    <div class="ma__breadcrumbs__label">
-      More in:
-    </div>
-    <ol class="ma__breadcrumbs__items">
-      <li class="ma__breadcrumbs__item"><a href="/">Home</a></li>
-      {% for link in breadcrumbs.links %}
-        <li class="ma__breadcrumbs__item"><a href="{{link.href}}">{{link.text}}</a></li>
-      {% endfor %}
-    </ol>
-  </nav>
-</section>
+<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
+  <nav aria-label="Breadcrumbs" class="ma__breadcrumbs__container">
+    <ol class="ma__breadcrumbs__items">
+      {% if breadcrumbs.links|length <= 2 %}
```

Comment:
```suggestion
      {% if breadcrumbs.links|length <= 2 %}
```
should it be <=3?

Reply:
no, because 2 is equal to the back caret + the home link, which means just the home link.